build.gradle

```diff
@@ -52,8 +52,6 @@ tasks.withType(ScalaCompile).configureEach {
             "-Xsource:3",                       // Scala 3 migration flag
             "-Ytasty-reader"                    // Scala 3 migration flag
     ]
-    compileScala.sourceCompatibility = 21
```

Comment:
If I understood correctly, this should not be set in combination with --release 21 (above)